### DISCUSSION TITLE
Ignore build for other branches

### DISCUSSION THREAD

**Discussion by DawnMD** [vercel/community#3643] [Help]
### Problem Description

We have a repo for eg `abced` and we deployed three instances of it using separate branches namely `main`, `staging` and `prod`. But when we push commit to `staging` for example, the preview build for `prod` and `main` is also running simultaneously. Can we change that functionality to only build the preview of the specific instant the commit is pushed to or a PR is opened?

**Comment by amyegan** [comment]
Hey @DawnMD. You can configure your project's Ignored Build Step to skip builds matching scenarios you choose. There are a few presets as well as the ability to completely customize a script. You can learn more at https://vercel.com/changelog/improved-experience-for-configuring-ignored-builds

**Reply by DawnMD** [reply]
Hey @amyegan 
I was checking out the option `Only build if there are changes` but I'm confused if the diffs being checked for the branch or only main to PR?
